docker/cache-redis-cluster.env

```diff
@@ -0,0 +1,5 @@
+CACHE_TYPE=redis
+REDIS_CONNECTION_TYPE=cluster
+REDIS_NODES=redis-node-0:6379,redis-node-1:6379,redis-node-2:6379,redis-node-3:6379,redis-node-4:6379,redis-node-5:6379
+REDIS_USE_DEFAULT_POOL_CONFIG=false
+REDIS_PASSWORD=bitnami
```

Comment:
Please replace the password with "thingsboard'. I understand that the bitnami is the default password and author of the image. But these are thingsboard deployment scripts.

Reply:
done